dev-docs/examples/multiformat-example.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: example
+title: Prebid.js Multiformat Example
+description: Prebid.js Multiformat Example
+top_nav_section: dev_docs
+nav_section: quick-start
+about:
+- Multiformat ads allow you to declare multiple media types on a single ad unit
+- Set up one ad unit that could show a banner, native, or outstream video ad
+- Any bidder that supports at least one of the listed media types can participate in the auction for that ad unit
+- For engineering setup instructions, see <a href="/dev-docs/show-multiformat-ads.html">Show Multiformat Ads</a>
+- For ad ops setup instructions, see <a href="/adops/setting-up-prebid-multiformat-in-dfp.html">Setting up Prebid Multiformat in DFP</a>
+jsfiddle_link: jsfiddle.net/prebid/mg81j0rw/3/embedded/html,result
```

Comment:
Remove reference to "http://mlane.devnxs.net/multiformat" in JSFiddle

Reply:
Thanks, updated in e78286a